src/pages/rockets.js

```diff
@@ -1,7 +1,15 @@
-const Rockets = () => {
+import React from 'react';
+
+const Rockets = ({rocket}) => {
+  console.log(rocket)
+
   return (
     <div>
-      <h1>Rockets Site Under Construction</h1>
+      <h1> {rocket.rocket_name} </h1>
+      <h3> {rocket.rocket_type} </h3>
+      <p>{rocket.success_rate_pct} % success rate</p>
+      <img src={rocket.flickr_images} width="200" />
+
```

Comment:
You should consider adding an "alt" attribute to the img tag. This is a good practice.

Reply:
correct thanks